### PR TITLE
PMM-10173  added and applied new tarbal install script

### PR DESCRIPTION
**Pull request by vyarosh**

### CODE REVIEW

**Comment by vyarosh** [comment]
- pmm2-client.yml: _https://pmm.cd.percona.com/blue/organizations/jenkins/package-testing/detail/package-testing/1335_
- pmm2-client_custom_path.yml
- pmm2-client_integration.yml: _https://pmm.cd.percona.com/blue/organizations/jenkins/package-testing/detail/package-testing/1331_
- pmm2-client_integration_auth_config.yml
- pmm2-client_integration_auth_register.yml
- pmm2-client_integration_auth_setup.yml
- pmm2-client_integration_custom_path.yml
- pmm2-client_integration_custom_port.yml
- 
- pmm2-client_upgrade.yml
- pmm2-client_integration_upgrade.yml: _https://pmm.cd.percona.com/blue/organizations/jenkins/package-testing/detail/package-testing/1335_
- pmm2-client_integration_upgrade_custom_path.yml
- pmm2-client_integration_upgrade_custom_port.yml

File: playbooks/pmm2-client_integration_custom_path.yml [hunk 1]
```diff
@@ -80,193 +49,61 @@
...
+
+#  - name: check if Metric End point with Auth works for Postgres Exporter
+#    shell: "{{item}}"
+#    with_items:
+#      - bash -x ../scripts/pmm2_check_metric.sh --service_name pgsql_{{ instance_ip }} --metric_name pg_up{collector=\"\custom_query.hr\"\} --pmm_server_ip {{pmm_server_address}} --agent_type postgres_exporter --metric_value 1
+#      - bash -x ../scripts/pmm2_check_metric.sh --service_name pgsql_{{ instance_ip }} --metric_name pg_up{collector=\"\custom_query.lr\"\} --pmm_server_ip {{pmm_server_address}} --agent_type postgres_exporter --metric_value 1
+#      - bash -x ../scripts/pmm2_check_metric.sh --service_name pgsql_{{ instance_ip }} --metric_name pg_up{collector=\"\custom_query.mr\"\} --pmm_server_ip {{pmm_server_address}} --agent_type postgres_exporter --metric_value 1
+#      - bash -x ../scripts/pmm2_check_metric.sh --service_name pgsql_{{ instance_ip }} --metric_name pg_up{collector=\"\exporter\"\} --pmm_server_ip {{pmm_server_address}} --agent_type postgres_exporter --metric_value 1
+#
+#  - name: check if Metric End point with Auth works for Postgres Exporter when service added via socket
```
Comment (puneet0191) [inline playbooks/pmm2-client_integration_custom_path.yml:95]: any reason these are commented?

File: playbooks/pmm2-client_integration_upgrade_custom_path.yml [hunk 2]
```diff
@@ -58,34 +26,10 @@
...
-    - pmm2-client
-    when: ansible_os_family == "RedHat"
-
-  - name: setup pmm2-client artifacts for custom path
-    command: "{{item}}"
-    with_items:
-      - "chmod +x /package-testing/scripts/pmm2_client_custom_path.sh"
-      - "/package-testing/scripts/pmm2_client_custom_path.sh"
+  - name: install old pmm2-client tarball to custom path
+    command: "../scripts/pmm2_client_install_tarball.sh -v 2.26.0 -p {{ custom_path }}"
```
Comment (puneet0191) [inline playbooks/pmm2-client_integration_upgrade_custom_path.yml:32]: why do you specifically install 2.26.0 here?

File: playbooks/pmm2-client_integration_custom_path.yml (hunk 1 above)
Comment (vyarosh) [reply playbooks/pmm2-client_integration_custom_path.yml:95]: Yes, these check were originally absent in this test. The has been extracted from refactored task. 
so I've added it comented with the question at line 85
`# there was no metric check and removing - add it?`

File: playbooks/pmm2-client_integration_upgrade_custom_path.yml (hunk 2 above)
Comment (vyarosh) [reply playbooks/pmm2-client_integration_upgrade_custom_path.yml]: that's a temporary solution while I'm working on obtaining release version. it's fixed now.

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-package-tests-matrix/detail/pmm2-package-tests-matrix/513/pipeline/ - matrix job with hardcoded pre-upgrade version. so it goes 2.28.0 >> 2.29.1

File: playbooks/pmm2-client_integration_upgrade_custom_path.yml (hunk 2 above)
Comment (vyarosh) [reply playbooks/pmm2-client_integration_upgrade_custom_path.yml]: restored the hardcode until PMM-10247 is fixed

Comment (vyarosh) [reply playbooks/pmm2-client_integration_upgrade_custom_path.yml]: this way we will have ability to run upgrade tests to RC version

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-package-tests-matrix/detail/pmm2-package-tests-matrix/521 - last test failed on cleanup.
Please note, that I'm leaving hardcoded old version for upgrade tests to have ability to run test until PMM-10247 will be fixed

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-package-tests-matrix/detail/pmm2-package-tests-matrix/553/pipeline/

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-package-tests-matrix/detail/pmm2-package-tests-matrix/587/pipeline/

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**